ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/package.scala

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.appendonlydao
+
+import akka.stream.scaladsl.Source
+import anorm.{BatchSql, NamedParameter}
+import com.daml.lf.transaction.Node.KeyWithMaintainers
+
+/** Type aliases used throughout the package
+  */
+package object events {
+
+  type SqlSequence[A] = SqlSequence.T[A]
+
+  import com.daml.lf.value.{Value => lfval}
+  type ContractId = lfval.ContractId
```

Comment:
Any particular reason why these members are no longer `private[events]`

Reply:
yes, these type helpers are quite useful for the new indexer code as well, since they simplify greatly complicated types.
In order to be able to use them outside of this package I just simply made them public.
As for me I wouldn't have obligations to let them stay public, but if we want to be more strict in hiding stuff, I am happy to add here a TODO to revisit visibility here.
What do you think, should I add the TODO?

Reply:
TODO sounds fine